docs/content/preview/yugabyte-voyager/voyager-troubleshoot.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Troubleshoot YugabyteDB Voyager
+headerTitle: Troubleshoot
+linkTitle: Troubleshoot
+description: Troubleshoot issues in YugabyteDB Voyager.
+headcontent: Diagnose and troubleshoot issues when migrating to YugabyteDB Voyager
+menu:
+  preview_yugabyte-voyager:
+    identifier: voyager-troubleshoot
+    parent: yugabytedb-voyager
+    weight: 105
+type: docs
+---
+
+## Schema migration
+
+### Invalid constraint type with PostgreSQL 17
+
+When running the [assess migration](../reference/assess-migration/#assess-migration) or [export schema](../reference/schema-migration/export-schema/) commands on a PostgreSQL `17.0–17.2` source database, you may see the following error:
+
+```output
+pg_dump: error: query failed: ERROR:  invalid constraint type "n"
+pg_dump: detail: Query was: EXECUTE getDomainConstraints('16452')
+```
+
+This error happens if the source database contains domain objects with NOT NULL constraints.
+A bug in PostgreSQL versions 17.0–17.2 causes `pg_get_constraintdef()` to not interpret NOT NULL domain constraints, leading to failure in `pg_dump` version 17.6 when using this function to fetch schema information.
+
+Resolution: You can choose from one of the two options as follows:
+
+- Use an older `pg_dump` version.
+
+    1. Install a `pg_dump` version 17.5 or earlier (recommended to use a version same as your PostgreSQL version).
```

Comment:
This doesn't make any sense - if the problem is with 17.2, do I use the same version (recommended?) or earlier version? And how is 17.5 earlier than 17.0-17.2?

Reply:
There are two things: one is the PG server version, and the other is the pg_dump version, the issue is with PG server 17.0- 17.2 and pg_dump version 17.6, so the resolution is to use pg_dump 17.5 or older with PG 17.0-17.2 but recommended is to use pg_dump 17.2 with PG version 17.2 that will be the best case.

Reply:
Okay, I expanded this section a bit more based on your clarification.